personal_profile/resources/personal_profiles.py

```diff
@@ -5,31 +5,31 @@
         'Bill Gates',
         'Im a co-founder of microsoft, and have been the richest guy in the world for a while...',
         datetime.date(1955, 10, 28),
```

Comment:
Why Do you use this personal profile? 
I think it should be moved to ORM and read every thing dynamically, please open an Issue, with link to this PR comments

Reply:
@Yarboa could you please explain further? I understood you mean we don't need to write the table itself, but do you mean we need to receive it from the user dynamically? That's of course what is going to be done but don't we need this table to check the data migrations work? Do you expect that inside of this file we will have a function that creates the table from the web interface?

Reply:
This comment, is not for this PR, maybe consider opening an issue,
The PERSONAL_PROFILES is a list of lists.
Don't you think it could be data in DB, and insert this content as Data Migration?

Reply:
Opened an issue: #75